www/js/survey/time_insert.js

```diff
@@ -0,0 +1,22 @@
+var populateIdLABEL = function(time) {
+  var curriedPILABEL = function() {
+    populateIdLABEL(time);
+  };
```

Comment:
Why is this a new file that is checked in and how is it different from the `www/js/survey//external/time_insert.js`